### ISSUE TITLE
copy-paste friendly messages

### ISSUE THREAD

**Issue by LiNk-NY** [r-lib/rlang#1447]
Hi Lionel, @lionel- 

I hope you're well. 

When running devtools::document on my package, in a new instance, I get the error message:

```
Error in load_imports(path) : 
  The packages `BiocFileCache`, `RaggedExperiment`, and `TCGAutils` are
required.
Calls: suppressPackageStartupMessages ... load_imports -> deps_check_installed -> check_installed
```

It would be nice if the output were more friendly to copy-paste esp. outside interactive sessions, e.g., 
```r
  The required packages are not available:
'BiocFileCache', 'RaggedExperiment', 'TCGAutils'
```

## reprex

```r
rlang::check_installed(c("hmisc", "vcd", "mice"))
#' ℹ The packages `hmisc`, `vcd`, and `mice` are required.
```

Thank you !
-Marcel

**Comment by lionel-** [comment]
I changed the quotes to `"` but I kept the "and" for the last two packages. My reasoning is that it's not too hard to edit the "and" out and we preserve the sentence-like message for interactive sessions. Note that if specific versions are required, they will be included in the list inside parentheses, making it harder to copy/paste again. If not for this, I'd have used code syntax like `c("foo", "bar")` to format the list.
